canopen_chain_node/include/canopen_chain_node/chain_ros.h

```diff
@@ -13,8 +13,67 @@
 #include <boost/weak_ptr.hpp>
 #include <pluginlib/class_loader.h>
 
+#include <std_msgs/Int8.h>
+#include <std_msgs/Int16.h>
+#include <std_msgs/Int32.h>
+#include <std_msgs/Int64.h>
+#include <std_msgs/UInt8.h>
+#include <std_msgs/UInt16.h>
+#include <std_msgs/UInt32.h>
+#include <std_msgs/UInt64.h>
+
+#include <std_msgs/Float32.h>
+#include <std_msgs/Float64.h>
+
```

Comment:
```#include <std_msgs/String.h>``` is missing...

see https://github.com/ipa-fxm/ros_canopen/commit/fc05ca75806a8e55ea38d124c5b7bd4f4ae05b8b